<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1350</width>
    <height>551</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="minimumSize">
    <size>
     <width>800</width>
     <height>0</height>
    </size>
   </property>
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>2</x>
      <y>12</y>
      <width>821</width>
      <height>476</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QFrame" name="frame_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>781</width>
       <height>141</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QPushButton" name="showResult">
      <property name="geometry">
       <rect>
        <x>670</x>
        <y>30</y>
        <width>100</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>=</string>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>100</y>
        <width>501</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>631</width>
        <height>61</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_clear">
      <property name="geometry">
       <rect>
        <x>670</x>
        <y>80</y>
        <width>100</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>clear</string>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="doubleSpinBox_x">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>100</y>
        <width>62</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_x">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>100</y>
        <width>31</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>x  =</string>
      </property>
     </widget>
    </widget>
    <widget class="QFrame" name="frame_3">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>180</y>
       <width>401</width>
       <height>231</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QPushButton" name="pushButton_1">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>60</y>
        <width>100</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>1</string>
      </property>
      <attribute name="buttonGroup">
       <string notr="true">buttonGroup_text</string>
      </attribute>
     </widget>
     <widget class="QPushButton" name="pushButton_2">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>60</y>
        <width>100</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>2</string>
      </property>
      <attribute name="buttonGroup">
       <string notr="true">buttonGroup_text</string>
      </attribute>
     </widget>
     <widget class="QPushButton" name="pushButton_3">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>60</y>
        <width>100</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>3</string>
      </property>
      <attribute name="buttonGroup">
       <string notr="true">buttonGroup_text</string>
      </attribute>
     </widget>
     <widget class="QPushButton" name="pushButton_4">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>100</y>
        <width>100</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>4</string>
      </property>
      <attribute name="buttonGroup">
       <string notr="true">buttonGroup_text</string>
      </attribute>
     </widget>
     <widget class="QPushButton" name="pushButton_5">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>100</y>
        <width>100</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>5</string>
      </property>
      <attribute name="buttonGroup">
       <string notr="true">buttonGroup_text</string>
      </attribute>
     </widget>
     <widget class="QPushButton" name="pushButton_6">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>100</y>
        <width>100</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>6</string>
      </property>
      <attribute name="buttonGroup">
       <string notr="true">buttonGroup_text</string>
      </attribute>
     </widget>
     <widget class="QPushButton" name="pushButton_7">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>140</y>
        <width>100</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>7</string>
      </property>
      <attribute name="buttonGroup">
       <string notr="true">buttonGroup_text</string>
      </attribute>
     </widget>
     <widget class="QPushButton" name="pushButton_8">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>140</y>
        <width>100</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>8</string>
      </property>
      <attribute name="buttonGroup">
       <string notr="true">buttonGroup_text</string>
      </attribute>
     </widget>
     <widget class="QPushButton" name="pushButton_9">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>140</y>
        <width>100</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>9</string>
      </property>
      <attribute name="buttonGroup">
       <string notr="true">buttonGroup_text</string>
      </attribute>
     </widget>
     <widget class="QPushButton" name="pushButton_point">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>180</y>
        <width>100</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>.</string>
      </property>
      <attribute name="buttonGroup">
       <string notr="true">buttonGroup_text</string>
      </attribute>
     </widget>
     <widget class="QPushButton" name="pushButton_0">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>180</y>
        <width>100</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>0</string>
      </property>
      <attribute name="buttonGroup">
       <string notr="true">buttonGroup_text</string>
      </attribute>
     </widget>
     <widget class="QPushButton" name="pushButton_x">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>180</y>
        <width>100</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>x</string>
      </property>
      <attribute name="buttonGroup">
       <string notr="true">buttonGroup_text</string>
      </attribute>
     </widget>
     <widget class="QPushButton" name="pushButton_graph">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>10</y>
        <width>231</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Graph</string>
      </property>
     </widget>
    </widget>
    <widget class="QFrame" name="frame_4">
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>180</y>
       <width>381</width>
       <height>231</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QPushButton" name="pushButton_plus">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>100</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>+</string>
      </property>
      <attribute name="buttonGroup">
       <string notr="true">buttonGroup_text</string>
      </attribute>
     </widget>
     <widget class="QPushButton" name="pushButton_minus">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>20</y>
        <width>100</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>-</string>
      </property>
      <attribute name="buttonGroup">
       <string notr="true">buttonGroup_text</string>
      </attribute>
     </widget>
     <widget class="QPushButton" name="pushButton_sin">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>20</y>
        <width>100</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>sin</string>
      </property>
      <attribute name="buttonGroup">
       <string notr="true">buttonGroup_text</string>
      </attribute>
     </widget>
     <widget class="QPushButton" name="pushButton_mul">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>60</y>
        <width>100</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>*</string>
      </property>
      <attribute name="buttonGroup">
       <string notr="true">buttonGroup_text</string>
      </attribute>
     </widget>
     <widget class="QPushButton" name="pushButton_div">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>60</y>
        <width>100</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>/</string>
      </property>
      <attribute name="buttonGroup">
       <string notr="true">buttonGroup_text</string>
      </attribute>
     </widget>
     <widget class="QPushButton" name="pushButton_cos">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>60</y>
        <width>100</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>cos</string>
      </property>
      <attribute name="buttonGroup">
       <string notr="true">buttonGroup_text</string>
      </attribute>
     </widget>
     <widget class="QPushButton" name="pushButton_pow">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>100</y>
        <width>100</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>^</string>
      </property>
      <attribute name="buttonGroup">
       <string notr="true">buttonGroup_text</string>
      </attribute>
     </widget>
     <widget class="QPushButton" name="pushButton_mod">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>100</y>
        <width>100</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>mod</string>
      </property>
      <attribute name="buttonGroup">
       <string notr="true">buttonGroup_text</string>
      </attribute>
     </widget>
     <widget class="QPushButton" name="pushButton_tan">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>100</y>
        <width>100</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>tan</string>
      </property>
      <attribute name="buttonGroup">
       <string notr="true">buttonGroup_text</string>
      </attribute>
     </widget>
     <widget class="QPushButton" name="pushButton_asin">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>140</y>
        <width>100</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>asin</string>
      </property>
      <attribute name="buttonGroup">
       <string notr="true">buttonGroup_text</string>
      </attribute>
     </widget>
     <widget class="QPushButton" name="pushButton_acos">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>140</y>
        <width>100</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>acos</string>
      </property>
      <attribute name="buttonGroup">
       <string notr="true">buttonGroup_text</string>
      </attribute>
     </widget>
     <widget class="QPushButton" name="pushButton_atan">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>140</y>
        <width>100</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>atan</string>
      </property>
      <attribute name="buttonGroup">
       <string notr="true">buttonGroup_text</string>
      </attribute>
     </widget>
     <widget class="QPushButton" name="pushButton_opn">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>180</y>
        <width>100</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>(</string>
      </property>
      <attribute name="buttonGroup">
       <string notr="true">buttonGroup_text</string>
      </attribute>
     </widget>
     <widget class="QPushButton" name="pushButton_cls">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>180</y>
        <width>100</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>)</string>
      </property>
      <attribute name="buttonGroup">
       <string notr="true">buttonGroup_text</string>
      </attribute>
     </widget>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>390</x>
       <y>430</y>
       <width>58</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="frame_plus">
    <property name="geometry">
     <rect>
      <x>835</x>
      <y>12</y>
      <width>501</width>
      <height>476</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QFrame" name="frame_graph">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>481</width>
       <height>441</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QCustomPlot" name="widget" native="true">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>441</width>
        <height>371</height>
       </rect>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="doubleSpinBox_xmin">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>400</y>
        <width>62</width>
        <height>22</height>
       </rect>
      </property>
      <property name="minimum">
       <double>-1000000.000000000000000</double>
      </property>
      <property name="maximum">
       <double>1000000.000000000000000</double>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="doubleSpinBox_xmax">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>400</y>
        <width>62</width>
        <height>22</height>
       </rect>
      </property>
      <property name="minimum">
       <double>-1000000.000000000000000</double>
      </property>
      <property name="maximum">
       <double>1000000.000000000000000</double>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>400</y>
        <width>58</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>x min =</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>400</y>
        <width>58</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>x max = </string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1350</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>view/qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="buttonGroup_text">
   <property name="exclusive">
    <bool>true</bool>
   </property>
  </buttongroup>
  <buttongroup name="buttonGroup"/>
 </buttongroups>
</ui>
